src/logger/src/metrics.rs

```diff
@@ -221,6 +224,13 @@ impl Metric for SharedMetric {
     fn count(&self) -> usize {
         self.0.load(Ordering::Relaxed)
     }
+
+    #[cfg(target_arch = "x86_64")]
+    // It is necessary to also reset the old value in order to serialize later the correct one.
```

Comment:
The comment is a bit unclear - first things first, by `serialize` you refer to metric flush, not `persist`? If so, could you make the comment a bit more verbose (or point to `impl Serialize`) so that it's clear that after flush, the metric value will be the `delta` of `self.0 - self.1`?

Reply:
I updated the comment, hope it's clearer now.